dbbackup/management/commands/dbbackup.py

```diff
@@ -29,6 +29,8 @@ class Command(BaseDbBackupCommand):
         make_option("-s", "--servername", help="Specify server name to include in backup filename"),
         make_option("-z", "--compress", help="Compress the backup files", action="store_true", default=False),
         make_option("-e", "--encrypt", help="Encrypt the backup files", action="store_true", default=False),
+        make_option("-m", "--mongo", help="Backup a mongo db instead of sql. Uses MONGO_SETTINGS", action="store_true",
+                    default=False),
```

Comment:
If the django app uses django_mongo_engine, then yes it shouldn't be different,
However this feature was intended when you're using a MongoDb beside from your db used by Django say MySQL.
If you require people to have the mongo settings defined in the DATABASES django settings with django, it won't be really usable.
Another solution would be to create a different command line like mongodbbackup or nosqldbbackup.